src/Tizen.NUI.WindowSystem/src/public/EdgeDragData.cs

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright(c) 2023 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System;
+using System.ComponentModel;
+using System.Runtime.InteropServices;
+using static Tizen.NUI.WindowSystem.Interop.InputGesture;
+
+namespace Tizen.NUI.WindowSystem
+{
+    /// <summary>
+    /// Represents a wrapper class for a unmanaged gesture data handle.
+    /// </summary>
+    public class EdgeDragData : IDisposable
+    {
+        private InputGesture _gesture;
+        internal SafeGestureHandle _handle;
+        private bool disposed = false;
+
+        private static event EventHandler<EdgeDragEventArgs> _eventHandler;
+        private static EdgeDragCb _delegate;
+
+        /// <summary>
+        /// Creates a new EdgeDragData.
+        /// </summary>
+        public EdgeDragData(InputGesture gesture, int fingers, GestureEdge edge)
```

Comment:
could you add exception comment ? 
<exception cref=xxx> xxx </exception>
i.e) https://github.com/Samsung/TizenFX/blob/master/src/Tizen.NUI/src/public/Window/Window.cs#L103